src/app/containers/MostRead/utilities/processMostRead.test.js

```diff
@@ -197,17 +212,58 @@ describe('filterMostRead', () => {
       description: 'should return null when most read item title is missing',
       data: missingTitleData,
       numberOfItems: 1,
-      expectedReturn: null,
+      expectedReturn: [],
```

Comment:
Were we testing the wrong thing in here before?

Reply:
Yes, we were. Before we were passing in test data in the shape of a most read item instead of passing it in, in the shape of most read data that is returned from the endpoint.

Reply:
ah I see what's going on, before we were not passing a value for `lastRecordTimeStamp` which made it return null. 

However, this test is kinda deceiving as the processMostRead logic, wouldn't immediately return an empty array if there is a missing title/href value. It would just skip over that array item and go the next.

I'll update the test so that it's clearer now.